### ISSUE TITLE
Installation doesn't work

### ISSUE THREAD

**Issue by Pinguee** [jtessler/cargo-not#1]
Hey!

I'm currently working on a project as part of my internship at [ICube](https://icube.unistra.fr/) laboratory, I need to use Cargo-not but I wasn't able to get it working.

There's some issues I found : 
-Broken links
-Probably broken dependency management

Thanks for helping out!

**Comment by jtessler** [comment]
Hello!

Not surprised to hear things are broken -- it's been years since I've tried
building this project. I'll give it a try later, but can you provide more
details until then? What broken links have you found and what errors do you
see when you try to build it?

**Comment by Pinguee** [comment]
I get a 404 error when I try to make closure, I couldn't get any further.
I found these broken links in the make file :
CC_URL = http://dl.google.com/closure-compiler/compiler-latest.zip
CSS_URL = https://closure-stylesheets.googlecode.com/files/closure-stylesheets-20111230.jar

**Comment by Pinguee** [comment]
I manage to compile the projects by installing another version of google closure and by modifying some paths in the makefile
(I used the version from William Lin and I used a stylesheets jar I installed in my local library). I also changed the `cn/base.js` file (I added some goog.require). 
<img width="910" alt="Capture d’écran 2021-06-03 à 20 20 45" src="https://user-images.githubusercontent.com/73791865/120693358-32755700-c4a9-11eb-8d28-13854ee57be1.png">
<img width="474" alt="Capture d’écran 2021-06-03 à 20 22 05" src="https://user-images.githubusercontent.com/73791865/120693528-66507c80-c4a9-11eb-9007-5303ec438e9e.png">


So now when I do `make release` I have a lot warning like : 
```
cn/ui/toolbox.js:116:2: WARNING - [JSC_UNDEFINED_NAME] goog.base is never defined
  116|   goog.base(this, opt_domHelper);
         ^^^^^^^^^

cn/ui/toolbox.js:116:7: WARNING - [JSC_INEXISTENT_PROPERTY] Property base never defined on goog
  116|   goog.base(this, opt_domHelper);
              ^^^^

cn/ui/toolbox.js:133:0: WARNING - [JSC_DEPRECATED_PROP_REASON] Property inherits of type goog has been deprecated: Use ECMAScript class syntax instead.
  133| goog.inherits(cn.ui.CommandToolbox, cn.ui.Toolbox);
```
  
After that I get the html, css and javascript I need but I don't get anything when I launch `index.html`, I only get a blank screen. 
I'm kinda bad at explaining the things I tried to do and probably forgot some things but I hope you'll be able to help me.
